### ISSUE TITLE
Strange behavior of list-item-bullet-indent with nested unordered lists

### ISSUE THREAD

**Issue by N0rbert** [remarkjs/remark-lint#228]
### Subject of the issue

Strange behavior of list-item-bullet-indent with nested unordered lists.

### Your environment

*   **OS**: Ubuntu 16.04 LTS
*   **Packages**, **Env**:

        $ node --version
        v12.16.2
        $ remark --version
        remark: 12.0.0, remark-cli: 8.0.0

### Steps to reproduce

Have the following RMarkdown document:

```
# Unordered list

* 1st line, 1st level
* 2nd line, 1st level
    - 1st line, 2nd level
        + 1st line, 3rd level

```

The rendering [is available](https://johnmacfarlane.net/babelmark2/?text=%23+Unordered+list%0A%0A*+1st+line%2C+1st+level%0A*+2nd+line%2C+1st+level%0A++++-+1st+line%2C+2nd+level%0A++++++++%2B+1st+line%2C+3rd+level).

Run `remark` on it with the following `.remarkrc`:

```
{
  "plugins": [
    "remark-preset-lint-recommended",
    ["remark-lint-unordered-list-marker-style", false],
    ["remark-lint-list-item-spacing", false],
    ["remark-lint-list-item-indent", "space"],
    ["remark-lint-list-item-content-indent", true]
 ]
}
```

### Expected behaviour

Remark do not show any warnings about the file.

### Actual behaviour

Remark shows the warnings:

```
  5:5  warning  Incorrect indentation before bullet: remove 2 spaces  list-item-bullet-indent  remark-lint
  6:9  warning  Incorrect indentation before bullet: remove 2 spaces  list-item-bullet-indent  remark-lint
```

**Comment by ChristianMurphy** [comment]
@N0rbert the lint rule is working as expected.
The nested list is indented two levels in you example, if you want it to pass the lint rule change it to:
```markdown
# Unordered list

* 1st line, 1st level
* 2nd line, 1st level
  - 1st line, 2nd level
    + 1st line, 3rd level
```

or if you want to disable the rule

```json
["remark-lint-list-item-bullet-indent", false]
```

**Comment by wooorm** [comment]
The list bullets also seem to align on tab stops, but the paragraphs don’t, so you may also want to configure `remark-lint-list-item-indent` to stop at tab sizes, and indenting your paragraphs.
